dashboard/src/actions/operators.test.tsx

```diff
@@ -556,6 +556,26 @@ describe("listSubscriptions", () => {
         payload: subscriptions,
       },
     ];
+    await store.dispatch(operatorActions.listSubscriptions("default", "operators"));
+    expect(store.getActions()).toEqual(expectedActions);
+  });
+
+  it("list subscriptions in both the global and a single namespace", async () => {
```

Comment:
I missed the use case in which an operator is installed globally. In that case, the subscription is placed in the `operators` namespace so we need to check that as well to get the list of operators installed.